R/process.R

```diff
@@ -191,16 +191,18 @@ dataset_process <- function(filename_data_raw,
   # Record methods
   methods <- process_format_methods(metadata, dataset_id, sources, contributors)
 
+  excluded_taxa <- metadata$exclude_observations %>% util_list_to_df2()
```

Comment:
Is this meant to be added? Seems unrelated to everything else in this PR

Reply:
No, this wasn't intended - I had a great tangling of branches, trying to do two things at once, and this line slipped through. Meanwhile a different one was missed.